### DISCUSSION TITLE
On Editor Change Method

### DISCUSSION THREAD

**Discussion by mcone880** [GrapesJS/grapesjs#5138] [Q&A] [answered]
SO essentially I have a method that I need called anytime the editor is changed in any way. Is there some native method for this? I need a detailed explanation.

**Comment by mcone880** [comment]
So a bit more description, I am making a website that is essentially saving off projects onto another platform. I need a way to ensure that when the grapes-editor is changed in any way, that the json for the editor is saved off to the main application so I can save it. I already have a method that updates the state variable in charge of the json, but I need it called whenever the editor is updated in any way. Does anyone have any ideas how I might go about this?

**Comment by artf** [comment, marked as answer]
You can use `update` event
```js
editor.on('update', () => console.log('editor updated', editor.getProjectData()))
```

**Reply by mcone880** [reply]
Thanks a bunch man, really appreciate it.
